subsys/bluetooth/nrf_bt_adv/providers/gap_appearance.c

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <zephyr/sys/byteorder.h>
+#include <bluetooth/nrf_bt_adv.h>
+
+
+static int get_data(struct bt_data *ad, const struct nrf_bt_adv_state *state,
+		    struct nrf_bt_adv_feedback *fb)
+{
+	ARG_UNUSED(fb);
+
+	if (state->bond_cnt > 0) {
+		return -ENOENT;
+	}
+
+	static uint8_t data[sizeof(uint16_t)];
+
+	sys_put_le16(CONFIG_NRF_BT_ADV_PROV_GAP_APPEARANCE_VAL, data);
```

Comment:
Why do we want a separate kconfig for this? I suggest using the host GAP values:

```suggestion
	sys_put_le16(bt_get_appearance(), data);
```